packages/@lwc/engine/src/framework/__tests__/api.spec.ts

```diff
@@ -8,6 +8,7 @@ import * as api from '../api';
 import { createElement, LightningElement } from '../main';
 import { registerTemplate } from '../template';
 import { compileTemplate } from 'test-utils';
+import { registerDecorators } from '../decorators/register';
```

Comment:
We should be consistent and import `createElement`, `LightningElement`, and `registerDecorators` from the same path.